ro_help/hub/views.py

```diff
@@ -299,11 +299,10 @@ def get(self, request):
         response = []
 
         county = request.GET.get("county")
-        q = request.GET.get("q")
 
-        if county and q and len(q) > 0:
-            rows = City.objects.filter(county__iexact=county, city__istartswith=q).values_list("id", "city", named=True)
-            response = [{"v": row.id, "t": row.city} for row in rows]
+        if county:
+            rows = City.objects.filter(county__iexact=county).values_list("id", "city", named=True)
+            response = [{"id": row.id, "city": row.city} for row in rows]
```

Comment:
One of the reasons I made this query to require 2 parameters was to make it harder to scraping this data. Although, thinking about it, having a limit of 1 characters is not offering much more complexity as it would require only about 50*26 requests to get it all.

Another reason is that you would still load on average 250 cities for every county and that is still quite a big select box.